### DISCUSSION TITLE
[Feature request]:Compatibility of TOPAS with CentOS: Supported Versions?

### DISCUSSION THREAD

**Discussion by WKjie** [OpenTOPAS/OpenTOPAS#110] [Feature-Request]
### Feature request

Hi,

I hope this message finds you well. I am interested in running TOPAS on a CentOS-based system and wanted to inquire about compatibility. Could you please let me know if TOPAS is supported on CentOS, and if so, which specific versions are recommended or required for optimal performance?

Thank you for your help!

Best regards,

Wei Kaijie

### TOPAS category

Outcome model

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by tmasilela** [comment]
Hi Wei,

You should, in principle, be ale to follow the [Debian quickStart guide](https://github.com/OpenTOPAS/OpenTOPAS/blob/main/OpenTOPAS_quickStart_Debian.md) and successfully install TOPAS on a CentOS system. However note that certain commands might be slightly different (see Isaac's response [here](https://github.com/OpenTOPAS/OpenTOPAS/discussions/45). Good luck with the installation!

All the best,
Thongchai

**Reply by WKjie** [reply]
Thank you very much for your explanation, I will give it a try.
